SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetConnection.h

```diff
@@ -41,6 +41,8 @@ class SPATIALGDK_API USpatialNetConnection : public UIpConnection
 	///////
 	// End NetConnection Interface
 
+	virtual bool IsReplayConnection() const override { return false; }
```

Comment:
Isn't this also in the NetConnection interface?

Reply:
Smart